integration-libs/epd-visualization/components/visual-picking/visual-picking-tab/product-filter/visual-picking-product-filter.module.ts

```diff
@@ -0,0 +1,30 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+import {
+  CmsConfig,
+  I18nModule,
+  provideDefaultConfig,
+  UrlModule,
+} from '@spartacus/core';
+import { IconModule } from '@spartacus/storefront';
+import { VisualPickingProductFilterComponent } from './visual-picking-product-filter.component';
+import { VisualPickingProductFilterService } from './visual-picking-product-filter.service';
+
+@NgModule({
+  imports: [CommonModule, FormsModule, IconModule, UrlModule, I18nModule],
+  providers: [
+    VisualPickingProductFilterService,
+    provideDefaultConfig(<CmsConfig>{
+      cmsComponents: {
+        VisualPickingProductFilterComponent: {
+          component: VisualPickingProductFilterComponent,
+        },
```

Comment:
Is this really a cms component? If not, we don't need this configuration.
Why I ask: I found in `epd-visualisation-root.module.ts` we declare only `VisualPickingTabComponent` as a cms component.

Although the redundant configuration doesn't break anything, it's confusing (as it suggests they might be real CMS components).

The same comment applies to a few other modules in this PR:
![image](https://user-images.githubusercontent.com/4001059/146752018-b285c3c0-cae1-4341-8944-547906cbab00.png)

Reply:
The VisualPickingTabComponent is the only actual CMSComponent. I can remove this from the others.